lib/backend/k8s/k8s.go

```diff
@@ -662,6 +671,40 @@ func (c *KubeClient) getPolicy(k model.PolicyKey) (*model.KVPair, error) {
 	}
 }
 
+// deletePolicy deletes a Policy. It takes care of appropriate conversions whether
+// the Policy is backed by a NetworkPolicy or a Global Network Policy.
+func (c *KubeClient) deletePolicy(d *model.KVPair) error {
+	k, ok := d.Key.(model.PolicyKey)
+	if !ok {
+		return goerrors.New("Incorrect key")
+	} else if k.Name == "" {
+		return goerrors.New("Missing policy name")
```

Comment:
Probably fine to have this check, but I _think_ this will be validated before it gets to us so we won't have to worry about this case.